django_project/ford3/tests/views/test_provider.py

```diff
@@ -0,0 +1,64 @@
+from django.urls import reverse
+from django.test import TestCase
+from ford3.tests.models.model_factories import ModelFactories
+
+
+class TestProvider(TestCase):
```

Comment:
add a model test instead

Reply:
remove test case is elimated. 
however, i'll like adding add and edit test cases for provider in this PR.
or, do you prefer having it in separate PR?